sample-lnd.conf

```diff
@@ -293,6 +293,15 @@
 ; The maximum number of incoming pending channels permitted per peer.
 ; maxpendingchannels=1
 
+; The maximum time until we delete a pending channel opening flow from memory.
```

Comment:
I wouldn't put test-only (or rather -mostly?) information in the public sample configuration. Assuming developers are clever enough to find those settings, I'd rather avoid confusing "plebs" with this. The defaults are fine, I guess?

Reply:
lemmie know what you think with the current approach.